pkg/registry/cache_test.go

```diff
@@ -59,9 +67,16 @@ func Test_CacheDependency(t *testing.T) {
 		}
 		for _, lib := range libs {
 			a.On("UpdateLib", lib.Name, lib).Return(nil)
+
+			var checker installedChecker
+			// checker.On("IsInstalled", pkg.Descriptor{
```

Comment:
Is this coming back in the future, or should it be deleted?

Reply:
The test may become more elaborate in the future - for now I'll take these out. 👍